lib/3scale/backend/async_redis/endpoint_helpers.rb

```diff
@@ -39,6 +60,10 @@ def format_ssl_params(ssl_params)
 
             updated_ssl_params
           end
+
+          def host_present?(host)
+            not host.to_s.strip.empty?
```

Comment:
not? :grimacing: I think in our code base we  have always used `!`
```suggestion
            !host.to_s.strip.empty?
```

Reply:
A more rails approach and more readable would be
```suggestion
            host.present?
```

Reply:
And actually no need for a separate method for this, you can just inline wherever :)